index.html

```diff
@@ -0,0 +1,156 @@
+      </section>
+    </header>
+
+    <section class="shadow">
+      <div id="program" class="main-program">
+        <div>
+          <h3 class="main-header">Main-program</h3>
+          <hr class="line" />
+        </div>
+        <div class="middle-content">
+          <div class="box-1">
+            <img src="./images/program_icon_01.png" alt="lecture" />
+            <h2 class="m-title">Lecture</h2>
+            <p class="box-para">
+              Listen to speakers from various brands about messages of sharing
+              and opening.
+            </p>
+          </div>
+          <div class="box-1">
+            <img src="./images/program_icon_02.png" alt="CC exhibiton" />
+            <h2 class="m-title">Exhibition</h2>
+            <p class="box-para">
+              Appreciate various creations applying CC license of artists,
+              organized from Wood and Words.
+            </p>
+          </div>
+          <div class="box-1">
+            <img src="./images/program_icon_03.png" alt="Forum" />
+            <h2 class="m-title">Forum</h2>
+            <p class="box-para">
+              Have the time to share your thoughts and opinions with experts for
+              the topic.
+            </p>
+          </div>
+          <div class="box-1">
+            <img src="./images/program_icon_04.png" alt="Workshop" />
+            <h2 class="m-title">Workshop</h2>
+            <p class="box-para">
+              Try creating your own work using open source license rather than
+              just watching.
+            </p>
+          </div>
+          <div class="box-1">
+            <img src="./images/program_icon_05.png" alt="CC ignite" />
+            <h2 class="m-title">CC ignite</h2>
+            <p class="box-para">
+              Get opportunities to network with CC affiliates around the world,
+              also after the summit.
+            </p>
+          </div>
+        </div>
+        <div>
+          <button id="join" class="button" type="button">
+            <a href="#">JOIN 2023 EXHIBITION</a>
+          </button>
+        </div>
+      </div>
+    </section>
+    <section id="news" class="features">
+      <div class="speakers-title">
+        <p class="fs-title">Featured Speakers</p>
+        <hr class="line" />
+      </div>
+
+      <div class="speaker"></div>
+      <button class="more-button"><span class="btn-1">More <i id="arrow-icon" class="fa-solid fa-chevron-down"></i></span><span class="hide btn-2">Less<i  id="arrow-icon" class="fa-solid fa-chevron-up"></i></span></button>
+    </section>
+
+    <div id="sponsor" class="partners">
+      <h3 id="p-title">Partners</h3>
+      <hr class="line" />
+      <div class="partners-c">
+        <img id="wood" src="./images/Logo 10.png" alt="Wood and words" />
+        <img id="toddle-care" src="./images/tc.jpg" alt="Toddle care" />
+      </div>
+    </div>
+
+    <footer class="footer">
+      <img src="./images/RO.jpeg" alt="" class="footer-img" />
```

Comment:
- [ ] Its best practice to include an `alt` on all images. please add one here to make your work professional 👏